src/AggregateFunctions/AggregateFunctionMinMaxAny.h

```diff
@@ -1120,6 +1124,8 @@ struct AggregateFunctionAnyHeavyData : Data
 template <typename Data>
 class AggregateFunctionsSingleValue final : public IAggregateFunctionDataHelper<Data, AggregateFunctionsSingleValue<Data>>
 {
+    static constexpr bool is_any = Data::is_any;
```

Comment:
I suppose `std::is_same_v<Data, AggregateFunctionAnyData>` would work as well, because if we would add such flags for other functions it would be lots of `is_<something>` flags. But for now it's ok to have just `is_any`

Reply:
No, it doesn't work because `AggregateFunctionAnyData` is itself a template so you can't use it like that,  or at least I didn't find a way (and I tried for quite a bit).

It's possible that I'll add further is_XXX (is_anylast?) in the future, but it's also possible that I'll break apart these functions to make it easier to introduce some SIMD (min, max...).

Reply:
If you know of a way to do it *at compile time* without using `is_XXX` that would be awesome.